README.md

```diff
@@ -2762,78 +2578,95 @@ Hazelcast offers the following ways for distributed query purposes:
+                }
 
-    virtual int getClassId() const {
-        return TYPE_ID;
-    }
+                static int32_t get_class_id() noexcept {
+                    return 3;
+                }
 
-    virtual void writePortable(serialization::portable_writer &writer) const {
-        writer.writeInt(id);
-        writer.writeUTF(&name);
-        writer.writeBoolean(active);
-        writer.writeDouble(salary);
-    }
+                static void write_data(const person &object, hazelcast::client::serialization::portable_writer &out) {
+                    out.write(object.name);
+                    out.write(object.male);
+                    out.write(object.age);
+                }
 
-    virtual void read_portable(serialization::portable_reader &reader) {
-        id = reader.readInt();
-        name = *reader.readUTF();
-        active = reader.readBoolean();
-        salary = reader.readDouble();
+                static person read_data(hazelcast::client::serialization::portable_reader &in) {
+                    return person{in.read<std::string>(), in.read<bool>(), in.read<int32_t>()};
+                }
+            };
+        }
     }
-
-private:
-    int32_t id;
-    std::string name;
-    bool active;
-    double salary;
-};
+}
 ```
 
-Note that `Employee` is implementing `Portable`. As portable types are not deserialized on the server side for querying, you don't need to implement its Java equivalent on the server side for querying.
+Note that `person` is being serialized with portable_serializer`. As portable types are not deserialized on the server side for querying, you don't need to implement its Java equivalent on the server side for querying.
 
 For the non-portable types, you need to implement its Java equivalent and its serializable factory on the server side for server to reconstitute the objects from binary formats. 
-In this case before starting the server, you need to compile the `Employee` and related factory classes with server's `CLASSPATH` and add them to the `user-lib` directory in the extracted `hazelcast-<version>.zip` (or `tar`). See the [Adding User Library to CLASSPATH section](#1212-adding-user-library-to-classpath).
+In this case before starting the server, you need to compile the `person` and related factory classes with server's `CLASSPATH` and add them to the `user-lib` directory in the extracted `hazelcast-<version>.zip` (or `tar`). See the [Adding User Library to CLASSPATH section](#1212-adding-user-library-to-classpath).
 
-> **NOTE: Querying with `Portable` object is faster as compared to `IdentifiedDataSerializable`.**
+> **NOTE: Querying with `portable_serializer` is faster as compared to `identified_data_serializer` .**
 
-#### 7.7.1.2. Querying by Combining Predicates with AND, OR, NOT
+#### 7.7.1.2. Querying by Combining Predicates with AND, OR
 
 You can combine predicates by using the `and_predicate`, `or_predicate` and `not_predicate` operators, as shown in the below example.
```

Comment:
I see that in the title we remove "NOT", so I am guessing in this paragraph the `not_predicate` should also be removed?

Reply:
put back the example for `not` predicate.